site/themes/template/layouts/partials/version-warning.html

```diff
@@ -0,0 +1,9 @@
+{{ if eq .CurrentSection.Params.version "development" }}
+{{ $latest_url := replace .Permalink .CurrentSection.Params.version .Site.Params.docs_latest | relURL }}
+<div role="alert">
```

Comment:
uuuh, cool!

at some point it might be cool to have some form of different background color
<img width="1002" alt="Screen Shot 2021-11-29 at 11 46 35 AM" src="https://user-images.githubusercontent.com/3574444/143908706-448ad8f1-95f9-4035-a1fd-933d5059503e.png">

Reply:
Yeah hence my CSS comment. We need to tackle our CSS - I want to use whatever is the most common theme across VMWare for the time being.